main.go

```diff
@@ -6,19 +6,24 @@ import (
 	"os"
 	"strings"
 
-	"github.com/skpr/go-skprconfig"
+	skprconfig "github.com/skpr/go-config"
 	extensionsv1beta1 "github.com/skpr/operator/pkg/apis/extensions/v1beta1"
 
 	"github.com/skpr/mail/internal/provider/mailhog"
 	"github.com/skpr/mail/internal/provider/ses"
 )
 
 func main() {
+	config, err := skprconfig.Load()
+	if err != nil && os.IsNotExist(err) {
```

Comment:
Do we actually care if `config.json` doesnt exist?

Reply:
Good point. If we accept all required params can be provided by env vars, then we shouldn't require it.